packages/graphql/tests/tck/directives/auth/arguments/where/interface-relationships/implementation-where.test.ts

```diff
@@ -168,12 +168,12 @@ describe("Cypher Auth Where", () => {
             WITH this
             CALL {
                 WITH this
-                MATCH (this)-[:HAS_CONTENT]->(this_Comment:Comment)
+                MATCH (this)-[thisthis0:HAS_CONTENT]->(this_Comment:\`Comment\`)
                 RETURN { __resolveType: \\"Comment\\" } AS content
                 UNION
                 WITH this
-                MATCH (this)-[:HAS_CONTENT]->(this_Post:Post)
-                WHERE (exists((this_Post)<-[:HAS_CONTENT]-(:\`User\`)) AND all(auth_this0 IN [(this_Post)<-[:HAS_CONTENT]-(auth_this0:\`User\`) | auth_this0] WHERE (auth_this0.id IS NOT NULL AND auth_this0.id = $this_Postauth_param0)))
+                MATCH (this)-[thisthis1:HAS_CONTENT]->(this_Post:\`Post\`)
+                WHERE (exists((this_Post:\`Post\`)<-[:HAS_CONTENT]-(:\`User\`)) AND all(thisthis2 IN [(this_Post:\`Post\`)<-[:HAS_CONTENT]-(thisthis2:\`User\`) | thisthis2] WHERE (thisthis2.id IS NOT NULL AND thisthis2.id = $thisparam0)))
```

Comment:
I know we're currently suffering from this bug with `EXISTS`, but I feel like we should shortly (or maybe even already) be in a position where we can move to using `EXISTS` clauses across the board! 🥳